backend/endpoints/rom.py

```diff
@@ -122,12 +125,12 @@ def get_roms(
     collection_id: int | None = None,
     virtual_collection_id: str | None = None,
     search_term: str | None = None,
-    limit: int | None = None,
-    offset: int | None = None,
+    limit: int = 72,
```

Comment:
According to the [docs](https://uriyyo-fastapi-pagination.netlify.app/learn/tutorial_user_guide/add_to_route/), we shouldn't need to specify `limit` and `offset` here, and can avoid manually instantiating `LimitOffsetParams`.

We can test if that's the case, and whether those params are still added to the OpenAPI spec.

Reply:
yep works!